<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAttenuatorWidget</class>
 <widget class="QWidget" name="QAttenuatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USB Attenuator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resrc/attenuator.qrc">
    <normaloff>:/icon/attenuator.png</normaloff>:/icon/attenuator.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="attenuationGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Attenuation</string>
     </property>
     <layout class="QHBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="attenuationSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>525</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="attenuationSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resrc/attenuator.qrc">
          <normaloff>:/icon/configure.png</normaloff>:/icon/configure.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="settingsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Device Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>&amp;Connect</string>
        </property>
        <property name="icon">
         <iconset resource="../resrc/attenuator.qrc">
          <normaloff>:/icon/view-refresh.png</normaloff>:/icon/view-refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="disconnectButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Disconnect</string>
        </property>
        <property name="icon">
         <iconset resource="../resrc/attenuator.qrc">
          <normaloff>:/icon/dialog-cancel.png</normaloff>:/icon/dialog-cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Re&amp;fresh</string>
        </property>
        <property name="icon">
         <iconset resource="../resrc/attenuator.qrc">
          <normaloff>:/icon/edit-redo.png</normaloff>:/icon/edit-redo.png</iconset>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QTableView" name="deviceTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>525</width>
          <height>100</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>175</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>70</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>attenuationSlider</tabstop>
  <tabstop>attenuationSpinBox</tabstop>
  <tabstop>settingsButton</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>disconnectButton</tabstop>
  <tabstop>refreshButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resrc/attenuator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>attenuationSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>attenuationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>attenuationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>attenuationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>settingsGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
